react/Menu/index.jsx

```diff
@@ -175,8 +181,10 @@ Menu.propTypes = {
   onSelectDisabled: PropTypes.func,
   /** Global Styles for MenuItems */
   itemsStyle: PropTypes.object,
-  /** if you need fixed menu */
-  popover: PropTypes.bool
+  /** If you need fixed menu */
+  popover: PropTypes.bool,
+  /** Whether the menu should be initially opened */
+  initialOpen: PropTypes.bool
```

Comment:
SelectBox has a `menuIsOpen` prop. Should we keep the same name?

Reply:
I don't think so, menuIsOpen controls the component but here we do not control it. It is just the initial value.